### PR TITLE
278-documentación proyecto front

### PR DESCRIPTION
**Pull request by claugp06**
In this PR, the documentation of the proyect for the frontend part has been created.

### CODE REVIEW

File: PPL/5-Proyect-Documentation-Frontend.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+- Actualmente, existe un log in social que funciona tanto por parte del backend así como el frontend pero no se muestra en la aplicación porque faltaba por crear los cuestionarios correspondiente de registro para un usaurio que utilize este método de incio de sesión. Al enterarnos de que la funcionalidad probablemente no fuera muy utilizada por los futuros clientes, no se le dió prioridad y nos centramos en otras partes que se consideraban de mayor importancia.
+
+- Para el futuro, nos encantaría poder automatizar los métodos de contactos. Por ejemplo educadores pueden actualmente ver la información ed la familia del estudiante, lo ideal es que el educador pudiera usar directamente la platarforma para esa comunicación.  Al igual nos gustaría incluir esta opción desde la perspectiva de la familia si se quiere poner en contacto con el educador de su hijo.
+
+- Añadir un dialogo con la información del educador de su hijo (igual que esta hecho en educadores que ve las familias pero a la inversa, las familias ven los datos de los educadores)
```
Comment (auroranavas) [inline PPL/5-Proyect-Documentation-Frontend.md:232]: Dar un poco de contexto, parece que estás hablando del hijo del educador. Poner algo como "Que las familias puedan ver los datos de contacto de los educadores de sus hijos".

File: PPL/5-Proyect-Documentation-Frontend.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+
+## Cosas para mejorar
+Somos autocríticos de nuestro propio trabajo sabiendo y siendo conscientes de que siempre habrá cosas que se pueda mejorar para hacer las más accesibles. No solo se tiene esta autoevaluación en cuenta para el apartado en el cual estamos si no también el feedback recogido sobre las pruebas de los usuarios pilotos. En general, la mayoría de ellos se tuvieron en cuenta y se corrigieron de manera inmediata ya que era errores considerables como no mostrar mensajes de error. Aunque cabe destacar que no todos se consideraron debido al tiempo y se decidió trabajar sobre aquellos más relevantes. A continuación se va a mostrar aspesctos abiertos a mejoras del software que actualmente tenemos desarrollado.
+
+- En el primer feedback recibido, se proporciono la sugerencia de que en las pantalla (iniciando sesión como administrador) de socios y educadores la creación de asambleas y el añadir educadores de manera respectiva podrían cambiarse a un botón en vez de tener que cambiar de pestaña. Este feedback se ha tenido en cuenta y por eso se indica en este apartado de mejoras.
```
Comment (auroranavas) [inline PPL/5-Proyect-Documentation-Frontend.md:240]: ¿El feedback se tuvo en cuenta o no? En la frase final dices que se ha tenido en cuenta pero está en este apartado de mejoras. Es contradictorio.

**Review by auroranavas — CHANGES_REQUESTED** [review]
I think you did a wonderful job putting all of this information together and I can tell you put a lot of effort 👏. Don't get scared by the number of comments! It's 90% typos. In general, check the correct styling of punctuation, for example, spaces after commas or periods.

File: PPL/5-Proyect-Documentation-Frontend.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+- En las pantallas relacionadas con la pantalla principal, las actividades de su página web actual incluye más opciones. En un futuro estaría bien volver a preguntar a la ONG sobre ello por si se quisiera incluir.
+
+
+## Cosas para mejorar
+Somos autocríticos de nuestro propio trabajo sabiendo y siendo conscientes de que siempre habrá cosas que se pueda mejorar para hacer las más accesibles. No solo se tiene esta autoevaluación en cuenta para el apartado en el cual estamos si no también el feedback recogido sobre las pruebas de los usuarios pilotos. En general, la mayoría de ellos se tuvieron en cuenta y se corrigieron de manera inmediata ya que era errores considerables como no mostrar mensajes de error. Aunque cabe destacar que el feedback dado como sugerencia no se consideró  debido al tiempo y se decidió trabajar sobre aquello más relevantes y que no funciaba como se esperaba. A continuación se va a mostrar aspesctos abiertos a mejoras del software que actualmente tenemos desarrollado.
```
Comment (auroranavas) [inline PPL/5-Proyect-Documentation-Frontend.md:240]: "... y que no **funcionaba** como se esperaba."

**Review by auroranavas — CHANGES_REQUESTED** [review]
Thank you for taking the feedback into account, however, there is still stuff you didn't correct. I unresolved every comment you missed. Also, the title of the file is in Spanglish, if it's in English it should be `5-Project-Documentation-Frontend`.

**Comment by claugp06** [comment]
> `5-Project-Documentation-Frontend`.

Most of them I had fixed them but I did not save the file, my bad. Now everything should be ok and if not just say so I can fix it

**Review by auroranavas — APPROVED** [review]
Thank you for taking the feedback into account. You did a really good job with this! 🙌

**Review by auroranavas — APPROVED** [review]

**Pull request merged**